dataans/src-tauri/src/dataans/error.rs

```diff
@@ -34,6 +34,12 @@ pub enum DataansError {
 
     #[error("Can not create an image from raw image data")]
     ImageFromRaw,
+
+    #[error("Incorrect file type, should be json or md: {0}")]
+    IncorrectFileType(String),
```

Comment:
1. I don't think we should support importing from the `.md` file. The MD files are not well-structured and it's easy to break their structure and import the data incorrectly. We support the md data export only to simplify data transfer to another note-taking app. Nothing more :)
2. I think the error variant name should be improved. Currently, it doesn't specify what file and what file type is expected.

```suggestion
    #[error("Incorrect import file type: only `json` is supported: {0}")]
    IncorrectImportFileType(String),
```

(you can change the variant name and error message as you want. the code above is only a suggestion)